requirements.txt

```diff
@@ -2,7 +2,7 @@ bz2file==0.98.*
 coloredlogs==10.0.*
 cycler==0.10.*
 Cython==0.29.*
-dipy==1.5.*
```

Comment:
Leftover from your local config ? Do we also want to revert to dipy 1.4 in this PR ? (I wouldn't mind as I also made the change locally)

Reply:
Oups, sorry. Will fix soon